### DISCUSSION TITLE
For  FAQ1, how to obtain term_1, term_2, term_3

### DISCUSSION THREAD

**Discussion by legend-L24** [b-mazur/asaf#2] [Q&A]
Hello, In FAQ1, you provide a figure to describe how to merge prob file. I understand macrosate, P_up, P_down but do not understand how to obtain term_1, term_2, term_3 in the final prob file. I do not understand their meaning. I check the values in MOF-303 examples, all term_1, 2, 3 are very small. Do we need it? In my understanding, if we need transition matrix, we only need macrosate, P_up, P_down.

**Comment by bartoszmazurwro** [comment]
Hi,

the next terms in the Taylor expansion can be calculated from the potential energy recorded in the column `current_potential_energy_K`. These are not needed to calculate the macrostate probability distribution. They are only used for temperature extrapolation of the MPD. 

To better understand temperature extrapolation of the MPD and how to derive next order terms, I recommend reading these two papers: [10.1063/1.4975331](https://doi.org/10.1063/1.4975331) and [10.1063/1.4996759](https://doi.org/10.1063/1.4996759).
